plugin-server/src/ingestion/ingestion-consumer.test.ts

```diff
@@ -97,11 +98,31 @@ describe('IngestionConsumer', () => {
         site_url: 'us.posthog.com',
         now: fixedTime.toISO()!,
         event: '$pageview',
+        ...event,
```

Comment:
Move `...event` before `properties` so `properties` are not overriden, this changed a few snapshots